### DISCUSSION TITLE
Axios

### DISCUSSION THREAD

**Discussion by Liam-hi** [data-interaction-fed24/server#3] [Ideas]
Hi,

Anyone heard about Axios? Axios can be more convenient and easier to work with.

https://axios-http.com/docs/intro

**Comment by hens-virtuella-varld** [comment]
If I know correctly,
Axios is a library for making HTTP requests easy and saves a lot of work.
We can use Axios to make requests from both the browser and server sides.
It makes a request, parses the JSON, and adds it to whatever the response object includes.

Are we going to use it?
